wikipendium/wiki/templates/article.html

```diff
@@ -3,6 +3,21 @@
 
 {% block title %}{{ articleContent.get_full_title }} -{% endblock %}
 
+{% block meta %}
+<meta property=og:title content="{{ articleContent.get_full_title }}" />
+<meta property=og:type content=article />
+<meta property=og:image content=https://www.wikipendium.no/static/apple-touch/icon-114x114.png />
+<meta property=og:url content="{{ articleContent.get_absolute_url }}" />
```

Comment:
Yeah, kind of confusing that it doesn't, considering the name

https://docs.djangoproject.com/en/1.8/ref/models/instances/#get-absolute-url